Main.cs

```diff
@@ -66,6 +68,37 @@ private static bool OnToggle(UnityModManager.ModEntry modEntry, bool value)
             return true;
         }
 
+        private static void PatchPersistentJobs()
+        {
+            Type? persistentJobsModInteractionFeaturesType = GetPersistentJobsInteractionType();
```

Comment:
Please use `UnityModManager.FindMod()` to see if PersistentJobs is loaded before doing the reflection.

Reply:
Done